include/SQLiteCpp/SQLiteCppExport.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file    SQLiteCppExport.h
+ * @ingroup SQLiteCpp
+ * @brief   File with macros needed in the generation of Windows DLLs
+ *
+ *
+ * Copyright (c) 2012-2022 Sebastien Rombauts (sebastien.rombauts@gmail.com)
+ *
+ * Distributed under the MIT License (MIT) (See accompanying file LICENSE.txt
+ * or copy at http://opensource.org/licenses/MIT)
+ */
+
+#pragma once
+
+/*
+* #define SQLITECPP_COMPILE_DLL to compile a DLL under Windows
+* #define SQLITECPP_EXPORT to export symbols when creating the DLL, otherwise it defaults to importing symbols
+*/
+
+/* Windows DLL export/import */
+#if defined(WIN32) && defined(SQLITECPP_COMPILE_DLL)
+    #if SQLITECPP_DLL_EXPORT
+        #define DLL_API __declspec(dllexport)
```

Comment:
```suggestion
        #define SQLITECPP_DLL_API __declspec(dllexport)
```
I think it should be better if something like `SQLITECPP_API` instead of `DLL_API` in order to avoid accidental macro collisions with another libraries, but this should require a quick find and replace of it

Reply:
Yes I was wondering what would be the best naming convention here...happy to change this